CHANGELOG.md

```diff
@@ -10,21 +10,22 @@
     - `authStateManager.updateAuthState`
     - `authStateManager.subscribe`
     - `authStateManager.unsubscribe`
-  - Adds new methods in sdk scope:
-    - `sdk.loginRedirect`
+  - Adds new methods in sdk browser scope:
+    - `sdk.signInWithCredentials`
+    - `sdk.signInWithRedirect`
     - `sdk.getUser`
     - `sdk.getIdToken`
     - `sdk.getAccessToken`
-    - `sdk.handleAuthentication`
+    - `sdk.parseAndStoreTokensFromUrl`
```

Comment:
This function name seems a little unwieldy. Neither "parse" nor "Url" really give me much meaning, but they describe some mechanic. How about "storeTokensFromRedirect" or "getTokensFromRedirect"? It will be the matching book-end to "signInWithRedirect"